src/ReactiveDomain.PolicyStorage/Services/ApplicationLoadService.cs

```diff
@@ -0,0 +1,31 @@
+﻿using ReactiveDomain.Foundation;
+using ReactiveDomain.Messaging;
+using ReactiveDomain.Messaging.Messages;
+using ReactiveDomain.Policy.Domain;
+using ReactiveDomain.Policy.ReadModels;
+using System;
+
+namespace ReactiveDomain.PolicyStorage.Services
+{
+    public class ApplicationLoadService
```

Comment:
Is this class used anywhere? It looks incomplete.

Reply:
removed